client/classic/upe/settings/index.js

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
Forgot to mention - this is located under the `client/classic/upe` directory. I believe "classic" refers to "classic checkout", whereas "blocks" (a sibling directory) refers to "blocks checkout".
What do you think about moving this file from `client/classic/upe/settings/index.js` to `client/settings/index.js` so that it's not under the "classic checkout" scope?

Reply:
💯 I was just thinking about this - Settings should be independent of however UPE is displayed so it should be in its own directory.